src/sqlfluff/dialects/dialect_redshift.py

```diff
@@ -2380,3 +2383,19 @@ class FunctionSegment(ansi.FunctionSegment):
             ),
         ),
     )
+
+
+class FromClauseSegment(ansi.FromClauseSegment):
+    """Slightly modified version which allows for using brackets for content of FROM."""
+
+    match_grammar: Matchable = StartsWith(
+        "FROM",
+        terminator=Ref("FromClauseTerminatorGrammar"),
+        enforce_whitespace_preceding_terminator=True,
+    )
```

Comment:
I think this is the same as `ansi.FromClauseSegment`. Can we just not override `match_grammar`?

Reply:
Correct - we should only need to override `parse_grammar`.